my-app/src/App/pages/CoinPage/components/CoinDescription/CoinDescription.module.scss

```
@@ -0,0 +1,13 @@
+.description{
+  margin: 21px 17px 21px 17px;
+  h5{

```

во всех местах нужен пробел между селектором и фигурной скобкой
```suggestion
  h5 {
```